Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -411,7 +411,11 @@ class SettingsViewController: TableViewController {
             header: .title(Strings.security),
             rows: [
                 .boolRow(title: Strings.browserLock, detailText: Strings.browserLockDescription, option: Preferences.Privacy.lockWithPasscode, image: #imageLiteral(resourceName: "settings-passcode").template),
-                .boolRow(title: Strings.saveLogins, option: Preferences.General.saveLogins, image: #imageLiteral(resourceName: "settings-save-logins").template)
+                Row(text: "Logins & Passwords", selection: {
```

Comment:
translate, add `unowned self` to `selection` callback

Reply:
Fix at https://github.com/brave/brave-ios/pull/4124/commits/ded5065d24f539a7dee6bb8bcb40b93a2b70fe7b